x-pack/legacy/plugins/monitoring/public/components/alerts/alerts.js

```diff
@@ -31,6 +31,10 @@ const getColumns = (kbnUrl, scope) => [
     render: severity => {
       const severityIcon = mapSeverity(severity);
 
+      if (!severityIcon || !severityIcon.title) {
+        return null;
```

Comment:
I'm not sure we want to do this. If the tooltip isn't going to work because we don't have any content, we should still show the child, right? Perhaps we always show the child and if there is no context for the tooltip, we provide some dummy title, like `Tooltip unavailable` or something?

Reply:
You are absolutely right! Didn't notice the child there. I will fix all the ones with similar situations

Good catch 👍